README.md

```diff
@@ -12,37 +12,35 @@ Purpose
 
 Bio-Formats' primary purpose is to convert proprietary microscopy data into 
 an open standard called the OME data model, particularly into the OME-TIFF 
-file format. See the [statement of purpose](http://www.openmicroscopy.org/site/support/bio-formats/about/index.html) 
+file format. See the [statement of purpose](docs/sphinx/about/index.rst)
```

Comment:
While you're editing this page anyway, I'd be inclined to change the link text and following text here too - to just "see [About Bio-Formats] for further information."